content/en/docs/setup/install/multicluster/primary-remote_multi-network/index.md

```diff
@@ -167,6 +224,39 @@ spec:
 EOF
 {{< /text >}}
 
+Apply the configuration to `cluster2`:
+
+{{< text bash >}}
+$ istioctl install --context="${CTX_CLUSTER2}" -f cluster2.yaml
+{{< /text >}}
+
+{{< /tab >}}
+{{< tab name="Helm" category-value="helm" >}}
+
+Install Istio as remote in `cluster2` using standard Helm commands.
+
+First, install the `base` chart in `cluster2`:
+
+{{< text bash >}}
+$ helm install istio-base istio/base --version 1.24.0-alpha.0 -n istio-system --set profile=remote --set global.externalIstiod=true --kube-context "${CTX_CLUSTER2}"
+{{< /text >}}
+
+Then, install the `istiod` chart in `cluster2` with the following multi-cluster settings:
+
+{{< text bash >}}
+$ helm install istiod istio/istiod --version 1.24.0-alpha.0 -n istio-system --set profile=remote --set global.externalIstiod=true --set pilot.env.EXTERNAL_ISTIOD=true --set global.multiCluster.clusterName=cluster2 --set global.network=network2 --set istiodRemote.injectionPath=/inject/cluster/cluster2/net/network2  --set global.configCluster=true --set global.remotePilotAddress="${DISCOVERY_ADDRESS}" --kube-context "${CTX_CLUSTER2}"
+{{< /text >}}
+
+{{< warning >}}
+
+The `istiod-remote` Helm chart is only available from Istio release 1.23 onwards.
+
+{{< /warning >}}
```

Comment:
```suggestion
```

I would just remove this

1. It's no longer true (the chart no longer exists)
2. We didn't document its existence before this

Reply:
Forgot to remove this, will do.